plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ApplicationSourceListPage.java

```diff
@@ -308,6 +308,14 @@ private IObservableValue createLocalTemplateControls(TabFolder tabContainer,
 				.validatingBeforeSet(validator).to(BeanProperties
 						.value(IApplicationSourceListPageModel.PROPERTY_LOCAL_APP_SOURCE_FILENAME).observe(model))
 				.in(dbc);
+		
+		DataBindingUtils.addDisposableValueChangeListener(new IValueChangeListener() {
```

Comment:
works well and is expressive. Alternatively one could have written this as lambda:
```
		DataBindingUtils.addDisposableValueChangeListener((ValueChangeEvent event) -> {
			refreshWizardAfterLocationChange(localTemplateFilename.getValue().toString());
			asyncRedraw(parent);
		}
```